js/home.js

```diff
@@ -0,0 +1,9 @@
+console.log("rajat logged");
+
+const hamburger = document.querySelector(".square");
+
+const nav = document.querySelector(".nav-list");
+
+hamburger.addEventListener("click", () => {
+    nav.classList.toggle("slide");
+});
```

Comment:
add some on click animation to hamburger icon as well.
example rotate and change to X icon.

Reply:
This is not the priority but still, if you have time then please add this as well.